src/ocean/util/config/ConfigFiller.d

```diff
@@ -671,38 +669,31 @@ struct SetInfo ( T )
 *******************************************************************************/
 
 public template IsSupported ( T )
+{
+    const IsSupported = IsSupportedImpl!(T)();
+}
+
+private bool IsSupportedImpl ( T ) ( )
 {
     static if ( is(T : bool) )
-    {
-        const IsSupported = true;
-    }
+        return true;
     else static if ( isIntegerType!(T) || isRealType!(T) )
+        return true;
+    else static if ( isArrayType!(T) )
```

Comment:
If `static if (is(ElementTypeOf!(T) U))` works then you don't need to add the `IsSupportedImpl` function.